src/Blog.resi

```diff
@@ -1,10 +1,3 @@
-module Post: {
```

Comment:
Dedupe data. This is a bit too much. The pipeline for getting some blog post data is:

- Write in markdown in a specific format
- Parse it using frontmatter through Next
- Parse/transform it again using Js.Json through bs-json
- Transform it _again_ into our own little payload format
- Transform it AGAIN into this component format...

Composition for when you can't just solve the problem. We absolutely can just solve our problem here. Killing these steps is making the code so much more readable AND faster. I'm gonna kill more of these next.